nion/swift/test/ImageCanvasItem_test.py

```diff
@@ -291,6 +291,25 @@ def test_hand_tool_on_one_image_of_multiple_displays(self):
             document_controller.tool_mode = "hand"
             display_panel.display_canvas_item.simulate_press((100,125))
 
+    def test_zoom_tool_on_one_image_of_multiple_displays(self):
```

Comment:
Current test appears to be "doesn't crash". Is there a way to check it has done the zoom?

Reply:
This was replicating existing test methodology, I don't know if it can realistically replicate functionality.